inst/pages/98_exercises.qmd

```diff
@@ -1146,13 +1437,119 @@ Useful functions: scater::runMDS, runNMDS, transformAssay, ggplot, scater::plotR
 
 ### Heatmap visualization
 
-1. Load experimental dataset from mia.
-2. Visualize abundances with heatmap.
-3. Visualize abundances with heatmap after CLR + Z transformation.
+1. Load one of the datasets from mia and the miaViz and sechm packages.
+
+```{r}
+#| label: heatmap-ex1
+#| code-fold: true
+#| code-summary: "Show solution"
+#| eval: false
+
+library(mia)
+library(miaViz)
+library(sechm)
+
+data("GlobalPatterns", package = "mia")
+tse <- GlobalPatterns
+```
+
+2. Get the most prevalent features. Subset your TreeSe to those features and the first twenty taxa.
 
-See the OMA book for examples.
+```{r}
+#| label: heatmap-ex3
+#| code-fold: true
+#| code-summary: "Show solution"
+#| eval: false
 
+most_prevalent <-  names(head(getPrevalence(tse,sort=TRUE)))
 
+tse <- tse[most_prevalent, 1:20]
```

Comment:
There is subsetByPrevalent() that does this with one-liner

Reply:
Is this in mia Package? I get a function not found error using subsetByPrevalent.

Reply:
We just renamed it. The new version is not in Bioconductor yet, so install it with remotes::install_github